setup.py

```diff
@@ -0,0 +1,21 @@
+import setuptools
+
+with open("README.md", "r") as fh:
+    long_description = fh.read()
```

Comment:
Not sure this will work in real world. You need to add the README in the package as well